src/forms-opportunities/opportunityHandler.js

```diff
@@ -10,7 +10,77 @@
  * governing permissions and limitations under the License.
  */
 
-import { filterForms, generateOpptyData } from './utils.js';
+import { isNonEmptyArray, isNonEmptyObject } from '@adobe/spacecat-shared-utils';
+import { filterForms, generateOpptyData, isSearchForm } from './utils.js';
+
+function generateDefaultGuidance(scrapedData, oppoty) {
+  let largeForm = false;
+  let isVisibleATF = true;
```

Comment:
to make it readable I would have used isBelowTheFold : true so that the largeForm check and isBelowTheFold check is consistent